lib/chai/core/assertions.js

```diff
@@ -3013,19 +3025,14 @@ function closeTo(expected, delta, msg) {
     , flagMsg = flag(this, 'message')
     , ssfi = flag(this, 'ssfi');
 
-  new Assertion(obj, flagMsg, ssfi, true).is.a('number');
-  if (typeof expected !== 'number' || typeof delta !== 'number') {
-    flagMsg = flagMsg ? flagMsg + ': ' : '';
-    var deltaMessage = delta === undefined ? ", and a delta is required" : "";
```

Comment:
By removing this logic, we have lost some nice messages right? 

Where we used to specify that the delta was wrong, for example, we would now just complain that some value was meant to be numeric

Could we maybe pass custom messages to the new assertions to help with that? (On L3028)

Reply:
Yup, we can change the message back to match what we had. You can see in the diff where I updated all the messages for this message.

In short for tests like this:

```js
expect(1.5).to.be.approximately(1.0);
```

The message changed thusly:

```diff
< "the arguments to closeTo or approximately must be numbers, and a delta is required"
> "expected undefined to be numeric"
```

I thought this message would be more clear as it identifies which arguments are wrong and what is expected. But reading it back now it's not clear that there's a argument missing.

I'll try to see if I can add this back in a way that's more clear than what was here previously.